From: Yaroslav Halchenko <debian@onerussian.com>
Subject: avoid overflows for now
  Due to a bug in current numpy beta (or numexpr) tests would fail
  if operation on int leads to overflows (e.g. of pow operation).

  as a workaround for now -- avoid big ints

Origin:  Debian
Bug: https://github.com/pandas-dev/pandas/issues/15046
Last-Update: 2017-01-04

--- a/pandas/tests/test_expressions.py
+++ b/pandas/tests/test_expressions.py
@@ -40,7 +40,7 @@ _mixed2 = DataFrame({'A': _frame2['A'].c
                      'C': _frame2['C'].astype('int64'),
                      'D': _frame2['D'].astype('int32')})
 _integer = DataFrame(
-    np.random.randint(1, 100,
+    np.random.randint(1, 10,
                       size=(10001, 4)), columns=list('ABCD'), dtype='int64')
 _integer2 = DataFrame(np.random.randint(1, 100, size=(101, 4)),
                       columns=list('ABCD'), dtype='int64')
